src/chart/bar/BarView.js

```diff
@@ -205,6 +226,15 @@ export default echarts.extendChartView({
             })
             .execute();
 
+        var bgGroup = this._backgroundEl || (this._backgroundEl = new Group());
```

Comment:
Perhaps it's better to be named to `this._backgroundGroup` or `this._backgroundRoot`.

`_backgroundEl ` is easy to be confused with `_backgroundEls` when writing or debugging code.